src/routes/user.route.js

```diff
@@ -8,15 +8,23 @@ import session from 'express-session';
 import '../middleware/passport.js';
 import { LoginSchema, SignUpSchema } from '../models/validationSchema.js';
 import validate from '../middleware/validation';
-import { SignUp, Login, logout } from '../controllers/user.controller.js';
+import {
+  SignUp,
+  Login,
+  logout,
+  disableUserAccount,
+} from '../controllers/user.controller.js';
+
 import isAuthenticated from '../middleware/authentication.js';
 
 const router = Router();
 dotenv.config();
 
 router.use(cookieParser());
 router.use(bodyParser.urlencoded({ extended: false }));
-router.use(session({ secret: 'TOP_SECRET' }));
+router.use(
+  session({ secret: 'TOP_SECRET', saveUninitialized: false, resave: false })
```

Comment:
the secret is exposed into the codes 
u may need to store it in .env file

Reply:
changes requested done